src/main/java/use_case/EventScheduler.java

```diff
@@ -152,14 +153,16 @@ public int compareTo(TimeFrame o) {
  * Assumes that times do not overlap.
  */
 class SortAndSearch implements GapFinder {
+    @Override
     public LocalDateTime findTimeGap(List<TimeFrame> timeFramesToIgnore, Duration taskDuration) {
         Collections.sort(timeFramesToIgnore);
         TimeFrame first, second = null;
         for (TimeFrame t : timeFramesToIgnore) {
             first = second;
             second = t;
-            if (first != null && first.end.plus(taskDuration).isBefore(second.start))
+            if (first != null && first.end.plus(taskDuration).isBefore(second.start)) {
                 return first.end;
+            }
```

Comment:
reversely, don't add parenthesis on single line if statements. We're going by Clean Architecture and Uncle Bob is on a mission to rid the world of curly braces.

Reply:
the code styling plugin i used told me to put curly braces around - I'll change it back